contrib/bmc-snmp-proxy

```diff
@@ -379,22 +387,23 @@ esac
 
 case "$RETVAL" in
 	0|1) ;;
-	2) eval_gettext "${SCRIPT_NAME}: failed to read ${BMC_INFO} " 1>&2 ;;
-	3) eval_gettext "${SCRIPT_NAME}: failed to get proxy config." 1>&2 ;;
-	4) eval_gettext "${SCRIPT_NAME}: failed to set ${SNMPD_BMC_CONF}." 1>&2 ;;
-	5) eval_gettext "${SCRIPT_NAME}: failed to disable snmp proxy." 1>&2 ;;
-	6) eval_gettext "${SCRIPT_NAME}: failed to reload snmpd." 1>&2 ;;
-	7) eval_gettext "${SCRIPT_NAME}: failed to set snmpd config." 1>&2 ;;
-	8) eval_gettext "${SCRIPT_NAME}: failed to set IPMI alert dest." 1>&2 ;;
-	9) eval_gettext "${SCRIPT_NAME}: no free IPMI alert dest." 1>&2 ;;
-	10) eval_gettext "${SCRIPT_NAME}: failed to set IPMI PEF." 1>&2 ;;
-	11) eval_gettext "${SCRIPT_NAME}: failed to write snmptrapd.conf." 1>&2 ;;
-	12) eval_gettext "${SCRIPT_NAME}: snmpd not found." 1>&2 ;;
-	*) eval_gettext "${SCRIPT_NAME}: unknown error." 1>&2 ;;
+	2) ${OUTPUT} "${SCRIPT_NAME}: failed to read ${BMC_INFO} " 1>&2 ;;
```

Comment:
I believe `gettext` message ids must not include variable parts. Also, I don't see `TEXTDOMAIN` being set anywhere, so `gettext` won't probably find anything. Especially provided that there are no i18n files. Does anyone actually use these scripts? 

I know it wasn't your code initially, but since you've decided to make these scripts work, please make them actually work. That being said, I think that neither `"${SCRIPT_NAME}: "`, nor `"${BMC_INFO"` or whatever other variables must be left in the messages. For compatibility with various languages, I would change the output routine to `printf` or a `gettext`-obtained value with format specifiers. Like this:

```
output() {
    FMT=$(${OUTPUT} $1)
    shift
    printf "${FMT}" $*
}

...

output "Some fancy message about entity's %s value %d\n" ${ENTITY} ${VALUE}
```